2017/10/13 20:09:27 [error] 7#7: *3 open() "/var/www/robots.txt" failed (2: No such file or directory), client: 172.17.0.1, server: axis, request: "GET /robots.txt?1507925367044 HTTP/1.1", host: "axis"
2017/10/13 20:13:40 [error] 7#7: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 172.17.0.1, server: axis, request: "GET / HTTP/1.1", upstream: "fastcgi://172.17.0.3:9009", host: "axis"
2017/10/13 20:13:40 [error] 7#7: *4 open() "/var/www/robots.txt" failed (2: No such file or directory), client: 172.17.0.1, server: axis, request: "GET /robots.txt?1507925620251 HTTP/1.1", host: "axis"
2017/10/13 20:13:44 [error] 7#7: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 172.17.0.1, server: axis, request: "GET / HTTP/1.1", upstream: "fastcgi://172.17.0.3:9009", host: "axis"
2017/10/13 20:13:44 [error] 7#7: *4 open() "/var/www/robots.txt" failed (2: No such file or directory), client: 172.17.0.1, server: axis, request: "GET /robots.txt?1507925624890 HTTP/1.1", host: "axis"
2017/10/13 20:14:30 [error] 7#7: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 172.17.0.1, server: axis, request: "GET / HTTP/1.1", upstream: "fastcgi://172.17.0.3:9009", host: "axis"
2017/10/13 20:14:30 [error] 7#7: *4 open() "/var/www/robots.txt" failed (2: No such file or directory), client: 172.17.0.1, server: axis, request: "GET /robots.txt?1507925670080 HTTP/1.1", host: "axis"
2017/10/13 20:14:31 [error] 7#7: *4 connect() failed (111: Connection refused) while connecting to upstream, client: 172.17.0.1, server: axis, request: "GET / HTTP/1.1", upstream: "fastcgi://172.17.0.3:9009", host: "axis"
2017/10/13 20:14:31 [error] 7#7: *4 open() "/var/www/robots.txt" failed (2: No such file or directory), client: 172.17.0.1, server: axis, request: "GET /robots.txt?1507925671332 HTTP/1.1", host: "axis"
2017/10/13 20:18:15 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.17.0.1, server: axis, request: "GET / HTTP/1.1", upstream: "fastcgi://172.17.0.2:9009", host: "axis"
2017/10/13 20:18:15 [error] 6#6: *1 open() "/var/www/robots.txt" failed (2: No such file or directory), client: 172.17.0.1, server: axis, request: "GET /robots.txt?1507925895783 HTTP/1.1", host: "axis"
2017/10/13 20:18:16 [error] 6#6: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.17.0.1, server: axis, request: "GET / HTTP/1.1", upstream: "fastcgi://172.17.0.2:9009", host: "axis"
2017/10/13 20:18:16 [error] 6#6: *1 open() "/var/www/robots.txt" failed (2: No such file or directory), client: 172.17.0.1, server: axis, request: "GET /robots.txt?1507925896819 HTTP/1.1", host: "axis"
2017/10/13 20:21:02 [error] 7#7: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.17.0.1, server: axis, request: "GET / HTTP/1.1", upstream: "fastcgi://172.17.0.2:9009", host: "axis"
2017/10/13 20:21:02 [error] 7#7: *2 open() "/var/www/robots.txt" failed (2: No such file or directory), client: 172.17.0.1, server: axis, request: "GET /robots.txt?1507926062162 HTTP/1.1", host: "axis"
2017/10/13 20:23:10 [error] 7#7: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.17.0.1, server: axis, request: "GET / HTTP/1.1", upstream: "fastcgi://172.17.0.2:9009", host: "axis"
2017/10/13 20:23:10 [error] 7#7: *5 open() "/var/www/robots.txt" failed (2: No such file or directory), client: 172.17.0.1, server: axis, request: "GET /robots.txt?1507926190653 HTTP/1.1", host: "axis"
2017/10/13 20:23:11 [error] 7#7: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.17.0.1, server: axis, request: "GET / HTTP/1.1", upstream: "fastcgi://172.17.0.2:9009", host: "axis"
2017/10/13 20:23:11 [error] 7#7: *5 open() "/var/www/robots.txt" failed (2: No such file or directory), client: 172.17.0.1, server: axis, request: "GET /robots.txt?1507926191570 HTTP/1.1", host: "axis"
2017/10/13 20:23:19 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.17.0.1, server: axis, request: "GET / HTTP/1.1", upstream: "fastcgi://172.17.0.2:9009", host: "axis"
2017/10/13 20:23:19 [error] 7#7: *1 open() "/var/www/robots.txt" failed (2: No such file or directory), client: 172.17.0.1, server: axis, request: "GET /robots.txt?1507926199104 HTTP/1.1", host: "axis"
2017/10/13 20:23:19 [error] 7#7: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.17.0.1, server: axis, request: "GET / HTTP/1.1", upstream: "fastcgi://172.17.0.2:9009", host: "axis"
2017/10/13 20:23:19 [error] 7#7: *5 open() "/var/www/robots.txt" failed (2: No such file or directory), client: 172.17.0.1, server: axis, request: "GET /robots.txt?1507926199775 HTTP/1.1", host: "axis"
2017/10/13 20:23:20 [error] 7#7: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.17.0.1, server: axis, request: "GET / HTTP/1.1", upstream: "fastcgi://172.17.0.2:9009", host: "axis"
2017/10/13 20:23:20 [error] 7#7: *5 open() "/var/www/robots.txt" failed (2: No such file or directory), client: 172.17.0.1, server: axis, request: "GET /robots.txt?1507926200038 HTTP/1.1", host: "axis"
2017/10/13 20:25:31 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.17.0.1, server: axis, request: "GET / HTTP/1.1", upstream: "fastcgi://172.17.0.2:9009", host: "axis"
2017/10/13 20:25:31 [error] 7#7: *1 open() "/var/www/robots.txt" failed (2: No such file or directory), client: 172.17.0.1, server: axis, request: "GET /robots.txt?1507926331143 HTTP/1.1", host: "axis"
2017/10/13 20:25:32 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.17.0.1, server: axis, request: "GET / HTTP/1.1", upstream: "fastcgi://172.17.0.2:9009", host: "axis"
2017/10/13 20:25:32 [error] 7#7: *1 open() "/var/www/robots.txt" failed (2: No such file or directory), client: 172.17.0.1, server: axis, request: "GET /robots.txt?1507926332044 HTTP/1.1", host: "axis"
2017/10/13 20:29:01 [error] 7#7: *2 open() "/var/www/robots.txt" failed (2: No such file or directory), client: 172.17.0.1, server: axis, request: "GET /robots.txt?1507926541527 HTTP/1.1", host: "axis"
